src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/ConfigPersistenceManager.java

```diff
@@ -190,141 +191,116 @@ public static void setPostFeatureBranchCommits(String projectKey, Boolean checke
 		setValue(projectKey, "isPostFeatureBranchCommitsActivated", Boolean.toString(checked));
 	}
 
-	public static void setGitUris(String projectKey, String gitUris) {
-		setValue(projectKey, "gitUris", gitUris);
-	}
+	public static String getSemicolonStringFromList(List<String> list) {
```

Comment:
Please remove this custom handling and work with JSON strings instead. Please have a look on how the decision guidance configration is stored. Is there a way to convert a list of GitConfiguration objects into a JSON object?